src/nft/components/collection/Sweep.tsx

```diff
@@ -267,8 +225,11 @@ export const Sweep = ({ contractAddress, collectionStats, minPrice, maxPrice, sh
     jointCollections.forEach((asset) => {
       if (!asset.susFlag) {
         const isNFTX = asset.marketplace === Markets.NFTX
-        asset.priceInfo.ETHPrice = calcPoolPrice(asset, isNFTX ? counterNFTX : counterNFT20)
-        BigNumber.from(asset.priceInfo.ETHPrice).gte(0) && (isNFTX ? counterNFTX++ : counterNFT20++)
+        const poolPrice = calcPoolPrice(asset, isNFTX ? counterNFTX : counterNFT20)
```

Comment:
is the gte -> gt intentional?

Reply:
Yes, 0 is returned for non-pooled asset so we just use the base price in this case